app/index.html

```diff
@@ -144,7 +144,7 @@
 
                 </div>
                 <div class="col-md-6 content">
-                    <iframe width="560" height="315" src="https://www.youtube.com/embed/iLNTtqYfIs8?rel=0" frameborder="0" allowfullscreen></iframe>
+                    <iframe width="560" height="315" src="https://www.youtube.com/embed/h3dsbfaM5_U" frameborder="0" allowfullscreen></iframe>
```

Comment:
Could you enhance this change by adding the `?rel=0` back to the `src` URL? That means that Youtube will not put up the "related videos" stuff when the video ends, which is what we want for this presentation of the video.

You can make this change in your local branch then commit and push and it will automatically show up on this Pull Request. It's awesome!